<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>integ</class>
 <widget class="QMainWindow" name="integ">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>integ</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>f(X+1)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X-1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>266</x>
      <y>281</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>f(X+2)</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="X_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="minimum">
     <double>-100000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="X_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>320</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="minimum">
     <double>-100000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X+1</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="X1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="minimum">
     <double>-100000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>200</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>f(X)</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="X5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>0</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="minimum">
     <double>-1000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="FX5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>0</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="minimum">
     <double>-1000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>f(X-2)</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="FX2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="minimum">
     <double>-10000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>200</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X-2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>266</x>
      <y>80</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>f(X-3)</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="X2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="minimum">
     <double>-100000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="X_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>360</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="minimum">
     <double>-1000000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="FX_1">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>240</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="minimum">
     <double>-11000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="X4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="minimum">
     <double>-1000000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="FX_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>280</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="minimum">
     <double>-1000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>400</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>f(X+5)</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="FX">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="minimum">
     <double>-10000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>170</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>f(X-1)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>400</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X+5</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="X_1">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="minimum">
     <double>-100000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>320</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X+3</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X-5</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="FX1">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>160</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="minimum">
     <double>-1000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="FX_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>320</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="minimum">
     <double>-10000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="FX4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="minimum">
     <double>-10000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="X3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="minimum">
     <double>-100000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="FX3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>80</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="minimum">
     <double>-1000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>266</x>
      <y>40</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>f(X-4)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>360</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X+4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>266</x>
      <y>360</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>f(X+4)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X+2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X-4</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="X">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="minimum">
     <double>-10000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="X_5">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>400</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="minimum">
     <double>-1000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>0</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>f(X-5)</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="texto">
    <property name="geometry">
     <rect>
      <x>426</x>
      <y>30</y>
      <width>221</width>
      <height>141</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>X-3</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="FX_5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>400</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="minimum">
     <double>-10000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>1000000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="FX_4">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>360</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="minimum">
     <double>-100000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>10000000.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>320</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>f(X+3)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Borrar">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Borrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Calcular">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>250</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>h</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Salir">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="h">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>250</y>
      <width>62</width>
      <height>22</height>
     </rect>
    </property>
    <property name="decimals">
     <number>10</number>
    </property>
    <property name="minimum">
     <double>-100000000.000000000000000</double>
    </property>
    <property name="maximum">
     <double>100000000.000000000000000</double>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>776</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
